Jenkinsfile

```
@@ -129,6 +129,14 @@ spec:
       }
     }
 
+    // Teardown the resources created during the Create stage
+    stage('Teardown') {
+      steps {
+        container('k8s-node') {
+             sh 'USER=jenkins gcloud compute ssh jenkins-deploy-dev-infra@gke-tutorial-bastion --command \'scripts/teardown.sh\''
```

We should not need the USER= with the new Jenkins file ... let me merge that and you can rebase.

```suggestion
             sh 'gcloud compute ssh jenkins-deploy-dev-infra@gke-tutorial-bastion --command \'scripts/teardown.sh\''
```